### ISSUE TITLE
Aria-label is missing from drawer audio buttons

### ISSUE THREAD

**Issue by dancgray** [deltanet/adapt-audio#171]
Buttons in audioDrawer.hbs have not aria-label

**Comment by RobertPeek** [comment]
This has been fixed.
